python/ray/serve/handle.py

```diff
@@ -150,6 +177,15 @@ def __getattr__(self, name):
 
 
 class RayServeSyncHandle(RayServeHandle):
+    def _make_router(self) -> EndpointRouter:
+        # Delayed import because ray.serve.api depends on handles.
+        from ray.serve.api import create_or_get_async_loop_in_thread
```

Comment:
AFAICT `create_or_get_async_loop_in_thread` is only used for this -- just move it into this file?